### ISSUE TITLE
Transfer not working due to MAX_TX_ATTRIBUTES = 16

### ISSUE THREAD

**Issue by dauTT** [CityOfZion/neo-python#558]
### Current behavior
We currently have a limitation (MAX_TX_ATTRIBUTES = 16) of how many attributes can be stored in a transaction. ([code link](https://github.com/CityOfZion/neo-python/blob/master/neo/Core/TX/Transaction.py#L251))

This  limitation does not allow to transfer a token to the same address more then 7 times. For example 
Let us look at the following two transfers:

**Transfer 1.** 
```
neo> wallet tkn_send b9fbcff6e50fd381160b822207231233dd3c56c2 AXpNr3SDfLXbPHNdqxYeHK5cYpKMHZxMZ9 ANLeAruhbuBRyb8CFn5H31y9a2c84vpWb3 10                   
Used 2.807 Gas 

-----------------------------------------------------------
Will transfer 10.00000000 NXT2 from AXpNr3SDfLXbPHNdqxYeHK5cYpKMHZxMZ9 to ANLeAruhbuBRyb8CFn5H31y9a2c84vpWb3
Transfer fee: 0.0001 
-------------------------------------------------------------

[Password]> **********                                                                                                                                   
[I 180818 00:15:40 Transaction:615] Verifying transaction: b'1b819a04d8c1093a8f84a2eacb7a92319cd3bc56fc4c9daaa89a98746c451385' 
Relayed Tx: 1b819a04d8c1093a8f84a2eacb7a92319cd3bc56fc4c9daaa89a98746c451385 

neo> tx 1b819a04d8c1093a8f84a2eacb7a92319cd3bc56fc4c9daaa89a98746c451385                                                                                 
{
    "txid": "0x1b819a04d8c1093a8f84a2eacb7a92319cd3bc56fc4c9daaa89a98746c451385",
    "size": 324,
    "type": "InvocationTransaction",
    "version": 1,
    "attributes": [
        {
            "usage": 32,
            "data": "affb37f5fdb9c6fec48d9f0eee85af82950f9b4a"
        },
        {
            "usage": 32,
            "data": "affb37f5fdb9c6fec48d9f0eee85af82950f9b4a"
        }
    ],
    "vout": [
        {
            "n": 0,
            "asset": "0x602c79718b16e442de58778e148d0b1084e3b2dffd5de6b7b16cee7969282de7",
            "value": "999.9993",
            "address": "AXpNr3SDfLXbPHNdqxYeHK5cYpKMHZxMZ9"
        }
    ],
    "vin": [
        {
            "txid": "7ba1b565bfa200c5549ab64e6e4fb1af4179edc8b40357f73a6590a747894dd2",
            "vout": 0
        }
    ],
    "sys_fee": "0",
    "net_fee": "0.0001",
    "scripts": [
        {
            "invocation": "4077cb15f34c0e585d48411b4a5c617ef5b2764afffbc007216330b04adbdb191164aeb0bd1f908fc4088c6c22245486a92050612f02bee7c6c3cdd260fcf28b69",
            "verification": "2103509010956ddc5e263ecbbca3ba966bb4608d2127eaf4608e489295242952b773ac"
        }
    ],
    "script": "0400ca9a3b1448033b58ef547cbf54c8ee2f72a42d5b603c00af14affb37f5fdb9c6fec48d9f0eee85af82950f9b4a53c1087472616e7366657267c2563cdd3312230722820b1681d30fe5f6cffbb9",
    "gas": 0,
    "height": 9783,
    "unspents": [
        {
            "n": 0,
            "asset": "0x602c79718b16e442de58778e148d0b1084e3b2dffd5de6b7b16cee7969282de7",
            "value": "999.9993",
            "address": "AXpNr3SDfLXbPHNdqxYeHK5cYpKMHZxMZ9"
        }
    ]
}

```

**Transfer 2.**
```
neo> wallet tkn_send b9fbcff6e50fd381160b822207231233dd3c56c2 AXpNr3SDfLXbPHNdqxYeHK5cYpKMHZxMZ9 ANLeAruhbuBRyb8CFn5H31y9a2c84vpWb3 10                   
Used 2.807 Gas 

-----------------------------------------------------------
Will transfer 10.00000000 NXT2 from AXpNr3SDfLXbPHNdqxYeHK5cYpKMHZxMZ9 to ANLeAruhbuBRyb8CFn5H31y9a2c84vpWb3
Transfer fee: 0.0001 
-------------------------------------------------------------

[Password]> **********                                                                                                                                   
[I 180818 00:16:32 Transaction:615] Verifying transaction: b'92d5bda71d79145573e76d8b828d9450c084d84da28c9fdb8021e6be7d7dff93' 
Relayed Tx: 92d5bda71d79145573e76d8b828d9450c084d84da28c9fdb8021e6be7d7dff93 
neo> tx 92d5bda71d79145573e76d8b828d9450c084d84da28c9fdb8021e6be7d7dff93                                                                                 
{
    "txid": "0x92d5bda71d79145573e76d8b828d9450c084d84da28c9fdb8021e6be7d7dff93",
    "size": 366,
    "type": "InvocationTransaction",
    "version": 1,
    "attributes": [
        {
            "usage": 32,
            "data": "affb37f5fdb9c6fec48d9f0eee85af82950f9b4a"
        },
        {
            "usage": 32,
            "data": "affb37f5fdb9c6fec48d9f0eee85af82950f9b4a"
        },
        {
            "usage": 32,
            "data": "affb37f5fdb9c6fec48d9f0eee85af82950f9b4a"
        },
        {
            "usage": 32,
            "data": "affb37f5fdb9c6fec48d9f0eee85af82950f9b4a"
        }
    ],
    "vout": [
        {
            "n": 0,
            "asset": "0x602c79718b16e442de58778e148d0b1084e3b2dffd5de6b7b16cee7969282de7",
            "value": "999.9992",
            "address": "AXpNr3SDfLXbPHNdqxYeHK5cYpKMHZxMZ9"
        }
    ],
    "vin": [
        {
            "txid": "1b819a04d8c1093a8f84a2eacb7a92319cd3bc56fc4c9daaa89a98746c451385",
            "vout": 0
        }
    ],
    "sys_fee": "0",
    "net_fee": "0.0001",
    "scripts": [
        {
            "invocation": "40c51d2465a326f81644fad51b3c4f7614ed012bcb58214cdbb6622e38c5fef1cb21228f781d677c4ffbf94fa575f55a6f0ef9a29d867f91f9cb6042620eb0d2a0",
            "verification": "2103509010956ddc5e263ecbbca3ba966bb4608d2127eaf4608e489295242952b773ac"
        }
    ],
    "script": "0400ca9a3b1448033b58ef547cbf54c8ee2f72a42d5b603c00af14affb37f5fdb9c6fec48d9f0eee85af82950f9b4a53c1087472616e7366657267c2563cdd3312230722820b1681d30fe5f6cffbb9",
    "gas": 0,
    "height": 9785,
    "unspents": [
        {
            "n": 0,
            "asset": "0x602c79718b16e442de58778e148d0b1084e3b2dffd5de6b7b16cee7969282de7",
            "value": "999.9992",
            "address": "AXpNr3SDfLXbPHNdqxYeHK5cYpKMHZxMZ9"
        }
    ]
}
```
As we can see from the transfers above, **this attribute** : 
```
{
            "usage": 32,
            "data": "affb37f5fdb9c6fec48d9f0eee85af82950f9b4a"
        }
```
 **is repeated 2 times in transfer 1 and 4  times in transfer 2** . Following the same logic if we execute the same transfer 7 times we will get 14 identical attribute. In this way the 8th transfer will not work because it will have 16 attributes which violate the condition that number of attributes must be less then MAX_TX_ATTRIBUTES = 16: ([code link](https://github.com/CityOfZion/neo-python/blob/master/neo/Core/TX/Transaction.py#L573))
  
``` 
File "/home/dau/workspace/python/github.com/neo/venv2/lib/python3.6/site-packages/neo/Core/TX/Transaction.py", line 573, in SerializeUnsigned
    raise Exception("Cannot have more than %s transaction attributes" % self.MAX_TX_ATTRIBUTES)
Exception: Cannot have more than 16 transaction attributes
```

### Expected behavior

I would have expected to see only one attribute in each transfers. In this way, If I want,  I could execute the same transfer until my token is completely exhausted. In addition, the repetitions of identical attributes seem to be a bug rather then a feature. 

### How to reproduce

read explanation above. The example can be reproduced by using any NEP5 token.

### Your environment
- OS: Ubuntu 18.x
- neo-python version: v0.7.5
- Python version:  3.6.5

**Comment by dauTT** [comment]
Thanks ixje for the bug fix! :+1:

**Comment by ixje** [comment]
thank you for the clear description! it really does help

**Comment by localhuman** [comment]
Fixed in #560
